internal/utils/config.go

```diff
@@ -357,11 +358,14 @@ type (
 		MFAVerificationAttempt      hookConfig `toml:"mfa_verification_attempt"`
 		PasswordVerificationAttempt hookConfig `toml:"password_verification_attempt"`
 		CustomAccessToken           hookConfig `toml:"custom_access_token"`
+		SendSMS                     hookConfig `toml:"send_sms"`
+		SendEmail                   hookConfig `toml:"send_email"`
 	}
 
 	hookConfig struct {
-		Enabled bool   `toml:"enabled"`
-		URI     string `toml:"uri"`
+		Enabled         bool   `toml:"enabled"`
+		URI             string `toml:"uri"`
+		HTTPHookSecrets string `toml:"secrets"`
```

Comment:
```suggestion
		Secrets string `toml:"secrets"`
```

Shall we just name this field `Secrets` for consistency with toml tag?

Reply:
Yup. I’ll need to add an additional set of config vars with our latest changes as well.